<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Case Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="case_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>760</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_new_case">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>New Case</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_import_cases">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Cases</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete_cases">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Cases</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_cases">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Cases</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
